### PR TITLE
Provide `html` output with detected defects :globe_with_meridians: 

### PR DESCRIPTION
**Pull request by jamacku**

### CODE REVIEW

File: .github/workflows/differential-shellcheck.yml [hunk 1]
```diff
@@ -50,3 +50,11 @@ jobs:
           name: Differential ShellCheck SARIF
           path: ${{ steps.ShellCheck.outputs.sarif }}
           retention-days: 7
+
+      - if: ${{ always() }}
```
Comment (lzaoral) [inline .github/workflows/differential-shellcheck.yml:54]: nit: `${{ ... }}` for `if` is not necessary, e.g. like here: https://github.com/aufover/aufover-benchmark/blob/d2278c8569c59e44c088ff66e5e51d89ac60805e/.github/workflows/fedora.yml#L91

File: README.md [hunk 2]
```diff
@@ -346,6 +346,29 @@ Relative path to SARIF file containing detected defects. Example of use:
...
+### html
+
+Relative path to HTML file containing detected defects. Example of use:
+
+```yaml
+- id: ShellCheck
+  name: Differential ShellCheck
+  uses: redhat-plumbers-in-action/differential-shellcheck@v5
+
+- if: ${{ always() }}
```
Comment (lzaoral) [inline README.md:358]: ditto

File: docs/example.xhtml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.1//EN' 'http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd'>
+<html xmlns='http://www.w3.org/1999/xhtml'>
```
Comment (lzaoral) [inline docs/example.xhtml:3]: off-topic:  Could please create a `csdiff` issue to port `cshtml` to `html5` instead of `xhtml`.  Thank you!

**Review by lzaoral — APPROVED** [review]
LGTM!

**Comment by lzaoral** [comment]
@jamacku, sorry!  I have completely forgotten about this behaviour of mergify...

File: .github/workflows/differential-shellcheck.yml (hunk 1 above)
Comment (jamacku) [reply .github/workflows/differential-shellcheck.yml:54]: You are right; I'll drop it in the follow-up. Thank you

File: docs/example.xhtml (hunk 3 above)
Comment (jamacku) [reply docs/example.xhtml:3]: I was about to do just that :D

File: .github/workflows/differential-shellcheck.yml (hunk 1 above)
Comment (jamacku) [reply .github/workflows/differential-shellcheck.yml:54]: Addressed in:
- https://github.com/redhat-plumbers-in-action/differential-shellcheck/pull/401

**Pull request merged**